src/cclib/bridge/cclib2biopython.py

```diff
@@ -12,7 +12,7 @@
     # Fail silently for now.
     pass
 
-from cclib.parser.utils import PeriodicTable
+from ..parser.utils import PeriodicTable
```

Comment:
I think with this we should follow PEP8. Absolute imports are recommended, as they are usually more readable and tend to be better behaved: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Ok, I made all the changes for `src`, but I'm not sure what to do about imports within the test directory for relative calls.